.github/workflows/deploy-cloud-nodes.yml

```diff
@@ -0,0 +1,123 @@
+name: Deploy Cloud Nodes
+
+on:
+  workflow_dispatch:
+    inputs:
+      build-image:
+        description: 'Build docker image'
+        required: false
+        default: 'true'
+      deploy-iris:
+        description: 'Deploy Iris'
+        required: false
+        default: 'true'
+      deploy-anthony:
+        description: 'Deploy Anthony'
+        required: false
+        default: 'true'
+      deploy-brad:
+        description: 'Deploy Brad'
+        required: false
+        default: 'true'
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        if: github.event.inputs.build-image == 'true'
```

Comment:
Let's not worry about this now: but I wonder if we can evaluate this condition for the whole job, rather than each step?

Reply:
I tried to set the conditional for the whole job first. However, it caused the `build-image` job to get skipped if the conditional returned `false`.  This is an issue because it caused the `deploy-iris` job to get skipped because it has the following line, which is necessary to prevent the node from getting deployed before the `build-image` job completes.
```
  deploy-iris:
    needs: build-image
```